CHANGELOG.md

```diff
@@ -11,6 +11,12 @@
   will be lost. In this build we added notification in statusbar to signalize
   that the connection was lost and IDE must be restarted. In future IDE will try
   to automatically reconnect.
+- [Image visualizations][1367]. Visualizations for the Enso Image library. New
+  image visualization can display a JSON with an image encoded as a base64
+  string. Histogram visualization has a new feature allowing to display the
```

Comment:
The "JSON with an image encoded as a base64" is not understandable for the user: they do not know in what format visualizations receive data. They just see visualization attached to nodes returning some enso types. The exact format is defined by visualization themselfs, by setting proper preprocessor code. The useful information is: you can display Image type and string with image encoded in base64.